engines/dfc_provider/spec/requests/orders_spec.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+require_relative "../swagger_helper"
+
+RSpec.describe "Orders swagger", swagger_doc: "dfc.yaml" do
+  let(:user) { create(:oidc_user, id: 12_345) }
+  let(:enterprise) {
+    create(
+      :distributor_enterprise,
+      id: 10_000, owner: user, name: "Fred's Farm", description: "Beautiful",
+      address: build(:address, id: 40_000),
+    )
+  }
+  let(:product) {
+    create(
+      :base_product,
+      id: 90_000, name: "Apple", description: "Red",
+      variants: [variant],
+      primary_taxon: non_local_vegetable
+    )
+  }
+  let(:non_local_vegetable) {
+    build(
+      :taxon,
+      name: "Non Local Vegetable",
+      dfc_id: "https://github.com/datafoodconsortium/taxonomies/releases/latest/download/productTypes.rdf#non-local-vegetable"
+    )
+  }
+  let(:variant) { build(:base_variant, id: 10_001, unit_value: 1, sku: "AR", supplier: enterprise) }
+
+  before { login_as user }
+
+  path "/api/dfc/enterprises/{enterprise_id}/orders" do
+    parameter name: :enterprise_id, in: :path, type: :string
+
+    post "Create Order" do
+      response "404", "not found" do
+        context "without enterprises" do
+          let(:enterprise_id) { "default" }
+
+          run_test! {
+            expect(enterprise.distributed_orders).to be_empty
+          }
+        end
+
+        context "with unrelated enterprise" do
+          let(:enterprise_id) { create(:enterprise).id }
+
+          run_test! {
+            expect(enterprise.distributed_orders).to be_empty
+          }
+        end
+      end
+
+      response "201", "created" do
+        before { product }
+
+        context "with default enterprise id" do
+          let(:enterprise_id) { "default" }
+
+          run_test! {
+            expect(enterprise.distributed_orders.count).to eq 1
+          }
+        end
+
+        context "with given enterprise id" do
+          let(:enterprise_id) { 10_000 }
+
+          run_test! {
+            expect(enterprise.distributed_orders.count).to eq 1
+          }
+        end
```

Comment:
You may already know this, but Swagger documents only the last example in a group. It overrides the data of the previous ones.

Reply:
I didn't know!

Reply:
I think it's a bit unintuitive and caused a bit of confusion for me. Some releated discussion here but I'm not really sure enough to suggest the best solution:  https://github.com/rswag/rswag/issues/131